modules/node-app-service/locals.tf

```diff
@@ -2,10 +2,10 @@ locals {
   app_settings = merge(
     var.app_settings,
     {
-      DOCKER_ENABLE_CI                = true
-      DOCKER_REGISTRY_SERVER_PASSWORD = sensitive(data.azurerm_container_registry.acr.admin_password)
-      DOCKER_REGISTRY_SERVER_URL      = data.azurerm_container_registry.acr.login_server
-      DOCKER_REGISTRY_SERVER_USERNAME = data.azurerm_container_registry.acr.admin_username
+      DOCKER_ENABLE_CI         = true
+      docker_registry_password = sensitive(data.azurerm_container_registry.acr.admin_password)
+      docker_registry_url      = data.azurerm_container_registry.acr.login_server
+      docker_registry_username = data.azurerm_container_registry.acr.admin_username
     }
   )
 }

```

Comment:
https://registry.terraform.io/providers/hashicorp/azurerm/4.5.0/docs/resources/linux_web_app
NOTE:
docker_registry_url, docker_registry_username, and docker_registry_password replace the use of the app_settings values of DOCKER_REGISTRY_SERVER_URL, DOCKER_REGISTRY_SERVER_USERNAME and DOCKER_REGISTRY_SERVER_PASSWORD respectively, these values will be managed by the provider and should not be specified in the app_settings map.